src/Validator/Service/AbstractValidatorFactory.php

```diff
@@ -16,9 +16,9 @@
 use function sprintf;
 
 /**
- * Factory for creating NoObjectExists instances
+ * Base validator factory to be extended
```

Comment:
If you want a description for the class, I recommend to remove the extended part.

```suggestion
 * Base validator factory
```

1. It's abstract so it needs to be extended
2. If it's internal, it shouldn't invite to be extended outside of the DoctrineModule

Reply:
fixed with c190ce5